web_app/testSite.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Test</title>
+    <link rel="stylesheet" type="text/css" href="src/form.css">
+
+</head>
+<body>
+    <div id="task-modal" class="modal">
+        <div class="modal-content">
+        <span class="close">&times;</span>
+        <form id="frame">
+            <h1 id = "title">Create new Task</h1>
+            <div id="form-body">
+                <br>
+                <div class="params">
+                    <div class="desc">Task Name</div>   
+                    <div class="input-item">
+                        <input id="text-input" type="text"></input>
+                    </div>
+                </div>
+                <br>
+                <div class="params">
+                    <div class="desc">Task Description</div>   
+                    <div class="input-item">
+                        <textarea id="task-description" type="text" rows="5" cols="17"></textarea>
+                    </div>
+                </div>
+                <br>
+                <div class="params">
+                    <div class="desc">Completion Time</div>
+                    <div class="input-item">
+                        <input id="time" type="time"></input>
+                    </div>
+                </div>
+                <br>
+                <div class="params">
+                    <div class="desc">Completion Date</div>
+                    <div class="input-item">
+                        <input id = "date" type = "date"></input>
+                    </div>
+                </div>
+                <br>
+                <div class="params">
+                    <div class="desc">Color</div>
+                    <div class="input-item">
+                        <select name = "color" id = "color">
+                            <option value = "1">light</option>
+                            <option value = "2">medium</option>
+                            <option value = "3">heavy</option>
+                        </select>
+                    </div>
+                </div>
+                <br>
```

Comment:
I think if we make the form body use `display: flex` then set `flex-direction: column` we can avoid using these `<br>` tags